server/app.js

```diff
@@ -40,6 +42,8 @@ app.use("/users", auth, usersRouter);
 app.use("/login", loginRouter);
 app.use("/posts", postsRouter);
 app.use("/comments", commentRouter);
+app.use("/commentLikes", commentLikeRouter);
+app.use("/postLikes", postLikeRouter);
```

Comment:
I'd rather these routes were structured following the [convention](https://restfulapi.net/resource-naming/) for a REST api. Eg: `posts/{postId}/likes`

Low priority, so feel free to refactor now or merge now refactor later